openapi3gen/openapi3gen.go

```diff
@@ -104,6 +105,11 @@ func (g *Generator) generateWithoutSaving(parents []*jsoninfo.TypeInfo, t reflec
 		if a && b {
 			vs, err := g.generateSchemaRefFor(parents, v.Type)
 			if err != nil {
+				// TODO: this needs code review
```

Comment:
LGTM. What are you concerned with?

Reply:
Just wanted your eyes because I didn't feel confident I understood the purpose of the code block correctly.